cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2895,9 +2910,7 @@ function start-kube-addons {
   local -r src_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty"
   local -r dst_dir="/etc/kubernetes/addons"
 
-  if [[ "${KUBERNETES_TENANT_PARTITION:-false}" == "true" ]]; then
-    create-kubeconfig "addon-manager" ${ADDON_MANAGER_TOKEN} "${PROXY_RESERVED_IP}" "8888" "http"
```

Comment:
To tenant addons not need to talk to RP at all?

Reply:
current approach is to have all cluster components to talk to API server directly so proxy is removed. currently the add-ons listed on the cluster are note required to proxy server.
ybai2016@yb01-m2-secureport-kubemark-tp-1-master /etc/kubernetes/addons $ ls
dashboard  loadbalancing  metadata-proxy  node-problem-detector  rbac  storage-class